src/DynamoCoreWpf/UI/Prompts/EditWindow.xaml

```diff
@@ -4,18 +4,24 @@
         xmlns:ui="clr-namespace:Dynamo.UI"
         xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
         Title="{x:Static p:Resources.EditWindowTitle}" 
-        MinHeight="130" MaxHeight="200" Width="400">
+        MinHeight="130" Height="200" Width="400">
```

Comment:
@scottmitchell Do you think it would be worth to still define MaxHeight and MaxWidth so the dialog will not be infinitely big in any case?

Reply:
I don't see why that's necessary. It will open to 200x400, and if the user wants to get crazy resizing it after that, who are we to stop them?

Reply:
it's possible they will make it very large, then switch to a lower resolution - if the dialog does not resize like a window you can get into weird states where the dialog cannot be closed.

Reply:
@mjkkirschner @scottmitchell I would encourage you to explore code to limit dialog to screen size https://stackoverflow.com/questions/30238007/wpf-maxheight-to-screen-height or I'm fine we define a big max size you think is reasonable. 

The reason I recommend defining max size is
1. It gives us a boundary of expected behavior. You will see this often in web development that div max size defined, although the actual behavior varies between platform or display logic. But it is critical for devs to look at issue and tell instantly what should happen and what is not expected to happen.
2.  I have seen bugs before like what @mjkkirschner mentioned. It is also very common for people who use multiple monitors and switch between to get into the resolution issue.

Reply:
I have also checked the current bahavior of set value dialog, seems there is a max width of screen size already. So worth to check out behavior after your change there on max height

Reply:
@QilongTang @mjkkirschner I've defined MaxHeight and MaxWidth for the edit window